From 7a4168062fbab2e33ef9a42bca9f87a5921afac2 Mon Sep 17 00:00:00 2001
From: Gustavo Zacarias <gustavo@zacarias.com.ar>
Date: Tue, 9 Aug 2016 11:49:56 +0200
Subject: [PATCH] acinclude.m4: don't unset variables

Unsetting ac_cv_{func,lib}_* is bad, you can't feed the configure cache.
Terminate them with extreme prejudice.

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>
Signed-off-by: Adam Duskett <aduskett@gmail.com>
[aduskett@gmail.com: Update for 7.3.0]
---
 build/php.m4 | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/build/php.m4 b/build/php.m4
index e91ef988..9586c490 100644
--- a/build/php.m4
+++ b/build/php.m4
@@ -1568,8 +1568,6 @@ dnl PHP_CHECK_FUNC_LIB
 dnl
 AC_DEFUN([PHP_CHECK_FUNC_LIB],[
   ifelse($2,,:,[
-  unset ac_cv_lib_$2[]_$1
-  unset ac_cv_lib_$2[]___$1
   unset found
   AC_CHECK_LIB($2, $1, [found=yes], [
     AC_CHECK_LIB($2, __$1, [found=yes], [found=no])
@@ -1601,8 +1599,6 @@ dnl and as a fall back in the specified library. Defines HAVE_func and
 dnl HAVE_library if found and adds the library to LIBS.
 dnl
 AC_DEFUN([PHP_CHECK_FUNC],[
-  unset ac_cv_func_$1
-  unset ac_cv_func___$1
   unset found
 
   AC_CHECK_FUNC($1, [found=yes],[ AC_CHECK_FUNC(__$1,[found=yes],[found=no]) ])
-- 
2.7.4

